{"classFile":"ContextedRuntimeExceptionTest","className":"ContextedRuntimeExceptionTest","packages":"org.apache.commons.lang3.exception","methodInfos":[{"md5":"bFkVXkhQNFTFPuccMvzu4w==","methodName":"setUp","parameters":"[]"},{"md5":"+XksfPT7cbN4w647jS5yKQ==","methodName":"testContextedException","parameters":"[]"},{"md5":"0JcJQHb3/Vz1tPPH60roLw==","methodName":"testContextedExceptionString","parameters":"[]"},{"md5":"xZ3OgDBgfWhNA0cbyNYdGw==","methodName":"testContextedExceptionThrowable","parameters":"[]"},{"md5":"ToixFHxBgydRPzhJgPYm7w==","methodName":"testContextedExceptionStringThrowable","parameters":"[]"},{"md5":"/36TKnxwGbioaKyAtylusQ==","methodName":"testContextedExceptionStringThrowableContext","parameters":"[]"},{"md5":"S1xgKezBOZDT1vZ0fhqzgw==","methodName":"testNullExceptionPassing","parameters":"[]"}],"addLines":[27,29,30,31,32,33,36,37,38,44,45,47,52,53,54,61,62,63,71,72,73,81,82,83,84,85,86,88],"delLines":[21,22,24,30,32,33,34,37,38,39,45,46,48,53,54,55,62,63,64,72,73,74,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,143,144,145,146,147,148,150],"type":"MODIFY","path":"src/test/java/org/apache/commons/lang3/exception/ContextedRuntimeExceptionTest.java","pro_Commit":"6be2343397b8192fc1328da21962dea27a8b2422","pro_Date":"2011-07-07 07:54:57","sha1":"37ac67f19b12457281a94fb41356855554ae9b0f","file_Date":"2011-07-07 07:54:57","projectName":"commons-lang","commitID":"6be2343397b8192fc1328da21962dea27a8b2422"}